daft-derive/tests/fixtures/invalid/struct-field-not-diffable.stderr

```diff
@@ -0,0 +1,88 @@
+error[E0277]: the trait bound `NonDiffable: Diffable` is not satisfied
+ --> tests/fixtures/invalid/struct-field-not-diffable.rs:8:5
+  |
+8 |     b: NonDiffable,
+  |     ^ the trait `Diffable` is not implemented for `NonDiffable`
+  |
+  = help: the following other types implement trait `Diffable`:
+            &'a T
+            ()
+            (A, B)
+            (A, B, C)
+            (A, B, C, D)
+            (A, B, C, D, E)
+            (A, B, C, D, E, F)
+            (A, B, C, D, E, F, G)
+          and $N others
+
+error[E0277]: the trait bound `NonDiffable: Diffable` is not satisfied
+ --> tests/fixtures/invalid/struct-field-not-diffable.rs:5:10
+  |
+5 | #[derive(Diffable)]
+  |          ^^^^^^^^ the trait `Diffable` is not implemented for `NonDiffable`
```

Comment:
Wish I could squelch these errors -- sadly not possible at the moment due to https://github.com/rust-lang/rust/issues/48214.